tests/protocols/test_http.py

```diff
@@ -695,5 +695,8 @@ def app(scope):
 
     protocol = get_connected_protocol(app, protocol_cls, ws="wsproto")
     protocol.data_received(UPGRADE_REQUEST)
-    assert b"HTTP/1.1 400 Bad Request" in protocol.transport.buffer
-    assert b"Missing Sec-WebSocket-Version header" in protocol.transport.buffer
+    if b"HTTP/1.1 426 " in protocol.transport.buffer:
```

Comment:
What's this switch for?

Reply:
It's related to the `RemoteProtocolError ` and its `event_hint` value: it responds with 426 (Upgrade Required). Unfortunately, there is no body.

Reply:
> Unfortunately, there is no body.

Just to double check: Do you mean there's no status phrase? (Eg. `HTTP/1.1 426 Upgrade Required`) or there's no body?

Reply:
Either, I think.